appveyor.yml

```diff
@@ -0,0 +1,25 @@
+init:
+  - git config --global core.autocrlf true
+
+branches:
+  only:
+    - appveyor
+
+before_build:
+  - '"%VS140COMNTOOLS%VsDevCmd.bat"'
+  - BeforeBuild.cmd
+
+build:
+  project: src/Everything.sln
+  publish_nuget: true
```

Comment:
I'd remove the nuget stuff from the config file for now. We can the nuget stuff later.